contracts/cw1-subkeys/src/balance.rs

```diff
@@ -106,17 +110,29 @@ impl ops::Add<Balance> for Balance {
     }
 }
 
+impl ops::SubAssign<Coin> for Balance {
```

Comment:
Yeah, this has to panic (no result possible), so I didn't implement. Panics inside contracts (when run in vm) have no useful message. And we should give a sensible error message for "not enough funds".

I would just use the sub method and maybe add docs why

Reply:
I see. In fact I pushed this commit just to have a discussion about it. I'll revert it.